test/e2e/sourcery.js

```diff
@@ -1,49 +1,118 @@
+                }
+            })
+            .isVisible(login.getSpecificSelectUserRoleOption(userRole), function (result) {
+                if (result.status === c.ELEMENT_FOUND) {
+                    browser.click(login.getSpecificSelectUserRoleOption(userRole))
+                }
+            })
+            .assert.containsText(login.userRoleSelect, userRole)
+            .isVisible(common.submitButton, function (result) {
+                if (result.status === c.ELEMENT_FOUND) {
+                    browser
+                        .click(common.submitButton)
+                        .waitForElementVisible(timeLogging.loggedInUsersName);
+                }
+            })
+            .assert.containsText(timeLogging.loggedInUsersName, userUnderTest)
+            .waitForElementVisible(timeLogging.navbar)
+            .assert.containsText(timeLogging.navbar, timeLogging.navbarLinks)
+            .assert.containsText(timeLogging.navbarSelected, timeLogging.navbarSelectedValue)
+            .assert.cssProperty(timeLogging.navbarSelected, timeLogging.activeProperty, timeLogging.navbarActiveValue);
+    },
+
+    'Admin creates new task': function (browser) {
+        let ran = Math.random() * 100;
+        let ran1 = Math.random();
+        let taskName = ran * ran1;
+        let rate = 55;
+
+        browser
+            .url(browser.launchUrl + tasks.url)
+            .waitForElementVisible(common.pageTitle)
+            .isVisible(common.clickButton, function (result) {
+                if (result.status === c.ELEMENT_FOUND) {
+                    browser.click(common.clickButton)
+                }
+            })
+            .waitForElementVisible(common.pageTitle)
+            .assert.containsText(common.pageTitle, tasks.title)
+            .waitForElementVisible(tasks.taskNameField)
+            .setValue(tasks.taskNameField, taskName)
+            .waitForElementVisible(tasks.taskDesciptionField)
+            .setValue(tasks.taskDesciptionField, tasks.description)
+            .isVisible(tasks.billableField, function (result) {
+                if (result.status === c.ELEMENT_FOUND) {
+                    browser.click(tasks.billableField)
+                }
+            })
+            .isVisible(tasks.billableValue, function (result) {
+                if (result.status === c.ELEMENT_FOUND) {
+                    browser.click(tasks.billableValue)
+                }
+            })
+            .waitForElementVisible(tasks.hourlyRateField)
+            .clearValue(tasks.hourlyRateField)
+            .setValue(tasks.hourlyRateField, rate)
+            .isVisible(common.submitButton, function(result){
+                if(result.status === c.ELEMENT_FOUND){
+                    browser.click(common.submitButton)
+                }
+            })
+            .waitForElementVisible(tasks.successMessage)
+            .url(function (result){
+                browser.assert.notStrictEqual(result.value, tasks.createUrl);
```

Comment:
In this case .notStrictEqual works as just .notEquals. .notStrictEqual would make more sense to use for number value checking when number is in different type(e.g.integer and string). 
And also here you check if urls are different, you do not check 'create' word so you cannot check if new task was created or not. It would make more sense to check if url after /tasks has a number value that is bigger than 0 or something else. The best way to check is to go to all tasks and find the new created task.